modules/core/src/components/log-viewer/core-3d-viewer.js

```diff
@@ -142,8 +143,15 @@ export default class Core3DViewer extends PureComponent {
         styleParser: this._getStyleParser(nextProps)
       });
     }
-    if (this.props.frame !== nextProps.frame) {
+    if (this.props.frame !== nextProps.frame && nextProps.frame) {
       stats.get('frame-update').incrementCount();
+
+      const vehiclePosition = nextProps.frame.vehicleRelativeTransform.transformPoint([0, 0, 0]);
+      // Shift the entire world downward so that the vehicle is always on the ground (z = 0)
+      // TODO - support drone use case
+      this.setState({
+        worldTransform: new Matrix4().translate([0, 0, -vehiclePosition.z])
```

Comment:
`worldTransform` is a confusing name to me, does not indicate from which coord to which coord.

Reply:
I read this as the transform *to* the *world* reference frame no?  Just like *vehicleRelative* is the transform *to* the vehicleRelative reference frame.

Reply:
Yeah have both the from and two frame in the name does make things more clear.